mint-auditor/src/bin/main.rs

```diff
@@ -72,6 +73,19 @@ pub enum Command {
         #[clap(long, env = "MC_JSON")]
         json: bool,
     },
+    StartHttpServer {
+        /// Path to mint auditor db.
+        #[clap(long, parse(from_os_str), env = "MC_MINT_AUDITOR_DB")]
+        mint_auditor_db: PathBuf,
+
+        /// Oprtional port for HTTP server. Defualts to 8080
+        #[clap(long, env = "PORT")]
```

Comment:
I believe it's better if the default is here, then clap documents it for you.
See https://github.com/mobilecoinfoundation/mobilecoin/blob/master/mobilecoind-json/src/bin/main.rs#L25

Reply:
ahhh gotchya